src/Components/addItem.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { fb } from '../lib/firebase';
+import { FirestoreProvider } from 'react-firestore';
+
+class Item extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      itemname: '',
+    };
+  }
+  updateInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+  addItem = e => {
+    e.preventDefault();
+    const db = fb.firestore();
+    db.settings({
+      timestampsInSnapshots: true,
+    });
+    const itemRef = db.collection('items').add({
+      itemname: this.state.itemname,
+    });
+    this.setState({
+      itemname: '',
+    });
+  };
+  render() {
+    return (
+      <form onSubmit={this.addItem}>
```

Comment:
I think it's an interesting approach to use a `<form>` element to handle the submit! What were your thoughts around this vs. using an `onClick` handler for the `<button>` on line 40?